src/MassTransit/Monitoring/Instruments/StartedReceiveInstrument.cs

```diff
@@ -0,0 +1,42 @@
+#nullable enable
+namespace MassTransit.Monitoring
+{
+    using System;
+    using System.Diagnostics;
+    using System.Diagnostics.Metrics;
+
+
+    public struct StartedReceiveInstrument
+    {
+        readonly Counter<long> _inProgress;
```

Comment:
This feels _really_ expensive to me, a ton of copying references on every single call.